plugins/node/opentelemetry-instrumentation-dns/test/integrations/dns-slash-promises-lookup.test.ts

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  InMemorySpanExporter,
+  SimpleSpanProcessor,
+} from '@opentelemetry/sdk-trace-base';
+import * as assert from 'assert';
+import { NodeTracerProvider } from '@opentelemetry/sdk-trace-node';
+import { DnsInstrumentation } from '../../src';
+import type * as DnsPromisesType from 'dns/promises';
+import * as utils from '../utils/utils';
+import { assertSpan } from '../utils/assertSpan';
+import { SpanStatusCode } from '@opentelemetry/api';
+
+const memoryExporter = new InMemorySpanExporter();
+const provider = new NodeTracerProvider();
+provider.addSpanProcessor(new SimpleSpanProcessor(memoryExporter));
+
+const supportsPromises =
+  parseInt(process.versions.node.split('.')[0], 10) >= 15;
```

Comment:
why >=15?
Is it not [added in `10.6.0`?](https://nodejs.org/docs/latest-v14.x/api/dns.html#dns_dnspromises_lookup_hostname_options)

Reply:
`dns.promises` was added in 10.6.0, but `dns/promises` was [added in 15.0.0](https://nodejs.org/docs/latest-v15.x/api/dns.html#dns_dns_promises_api).

Reply:
right! i missed that :)